go/kbnm/install_host

```diff
@@ -47,7 +65,9 @@ install() {
 
   chmod +r "$target"
 
-  echo "Success: Installed Chrome NativeMessaging whitelist: "$host_path" for $host_name"
+  cat "$target"
```

Comment:
The old message was verbose and hard to parse, at least this way you know what got installed and easier to debug. This script is for "dev only", the consumer version will be embedded in our packaging, so I think this is better.